torch/csrc/jit/script/compiler.cpp

```diff
@@ -2241,8 +2273,35 @@ struct to_ir {
         auto values = getValues(ll.inputs(), /*maybe_unpack=*/true);
         return graph->insertNode(graph->createTuple(values))->output();
       } break;
+      case TK_DICT_LITERAL: {
+        auto dl = DictLiteral(tree);
+        std::vector<Value*> keys, values;
+        std::tie(keys, values) = getValues(
```

Comment:
All of the above stuff above dealing with a paired list of named values seems like huge overkill compared to:

```
auto keys = dl.key_inputs().tree()->trees();
auto values = dl.value_inputs().tree()->trees();
for(size_t i = 0; i < keys.size(); ++i) {
  keys.push_back(emitExpr(keys[i]));
  values.push_back(emitExpr(values[i]));
}
```